data/core/editor/help.cfg

```diff
@@ -483,7 +483,7 @@ There is a filter function to show only a subset of the available items — this
 
 " + _ "<header>Native</header>
 
-A map file consists of rows with comma separated terrain code strings. The only non-terrain information provided by the map syntax is the set of locations created by the <ref dst='editor_tool_starting'>Starting Locations Tool</ref>. The files can be edited with a general purpose text editor like notepad.
+A map file consists of rows with comma-separated terrain code strings. The only non-terrain information provided by the map syntax is the set of locations created by the <ref dst='editor_tool_starting'>Starting Locations Tool</ref>. The files can be edited with a general purpose text editor like Notepad.
```

Comment:
A bit of a nitpick, and obviously depends on what others think of it, but I think we should drop the Notepad reference altogether. Reasons: 1. Windows specific, 2. Notepad is outdated is some cases and may not support utf-8 properly.

Reply:
I agree that the reference is unnecessary for both of the reasons you stated, but usually I try to err on the side of changing as little as possible, particularly when it comes to strings (I did consider removing this). Sometimes I get a lot of flak for making what seems to me to be sensible changes but then I get not-very-helpful sentiments that I'm so-called 'bike-shedding', blah blah blah.

Reply:
I agree on removing the reference to Notepad, but it should probably be done separately from a general review.